src/AspNetCoreModuleV2/InProcessRequestHandler/inprocessapplication.cpp

```diff
@@ -468,6 +447,22 @@ IN_PROCESS_APPLICATION::ExecuteApplication(
         FINISHED_IF_FAILED(SetEnvironementVariablesOnWorkerProcess());
     }
 
+    LOG_INFO("Starting managed application");
+
+    if (m_pLoggerProvider == NULL)
+    {
+        FINISHED_IF_FAILED(hr = LoggingHelpers::CreateLoggingProvider(
```

Comment:
Should we prevent startup if we fail logging setup?

Reply:
CreateLoggingProvider just allocates the objects, it doesn't actually preform the redirection. So going to finished here makes sense.